modules/deployment/pages/replication.adoc

```diff
@@ -0,0 +1,50 @@
+= Replication
+
+Replication is used to automatically synchronize data between two and more directory server instances.
+The _dsreplication_ command works over SSL and uses the administration connector.
+The command can be also used in interactive mode by not providing any attributes to it.
+
+== Enable replication
+
+First enable replication by using the dsreplication subcommand _enable_:
+
+IMPORTANT: These variables have to be adjusted based on your environment settings:
+
+* $ADMIN_UID, $ADMIN_PASSWORD: These options refer to the Global Administrator, which exists in the _cn=Administrators, cn=admin data_ subtree.
+You can take advantage of the _ldapsearch_ command to view the Global Administrator entry:
+[,console]
+----
+ldapsearch --hostname $HOSTNAME1 --port $PORT1 --bindDN "cn=Directory Manager" --bindPassword $PASSWORD \
+        --useSSL --baseDN "cn=Administrators,cn=admin data" --searchScope sub "(objectclass=*)"
+----
+* $HOSTNAME1: hostname or IP of the first server whose contents will be replicated
+* $HOSTNAME2: hostname or IP of the second server
+* $PORT_NUMBER1: Port number of the first server whose contents will be replicated
+* $PORT_NUMBER2: Port number of the second server
+* $PASSWORD: Password of the corresponding server
+* $REPLICATION_PORT: Replication port number
+
+[,console]
+----
+./dsreplication enable --adminUID $ADMIN_UID --adminPassword $ADMIN_PASSWORD --trustAll --no-prompt --baseDN dc=example,dc=com \
```

Comment:
The command seems correct but I don't like the mixing of different variables for `port1` and `port2` and then same variable for `replicationPort1` and `replicationPort2` (and bind passwords).

Anyway I don't have any suggestions on how to format this.